### ISSUE TITLE
Chapter 11: Exercise 6 & 7

### ISSUE THREAD

**Issue by vvronskyFX** [jimmysong/programmingbitcoin#254]
I'm getting an error here when I run the test saying that "NameError: name 'merkleblock' is not defined"

Here's my output:
![image](https://user-images.githubusercontent.com/64465297/187837475-110d90e9-dd14-4892-a38f-59fbe650a6a6.png)

Here's my code:

class MerkleBlock:
![image](https://user-images.githubusercontent.com/64465297/187837678-8059ce26-a67e-4a99-999e-bd0cd686bad6.png)

Below, I have the 2 solutions to the two exercises respectively:
![image](https://user-images.githubusercontent.com/64465297/187837849-36607d42-e7f9-4978-90dd-1ac5b3ca74ec.png)

I matched it up with the original book solutions and it looks correct from what I can tell.

I tried "MerkleBlock" instead of "merkleblock" and I'm still getting the NameError. 

Has anyone encountered this? Any solutions?

**Comment by vvronskyFX** [comment]
Still stuck with no solution yet. @jimmysong would you have any idea to fix this?

**Comment by alexbarron** [comment]
Yes, I hit the same issue. I resolved it by adding `import merkleblock` to the top cell in the notebook that loads all the dependencies. 

```
############## PLEASE RUN THIS CELL FIRST! ###################

# import everything and define a test runner function
from importlib import reload
from helper import run
import block
import ecc
import helper
import network
import script
import tx
import merkleblock
```
